Cargo.toml

```diff
@@ -18,7 +18,14 @@ default = ["std"]
 std = []
 
 [dev-dependencies]
-pico-sys = "0.0.1"
+criterion = "0.3.5"
+
+[lib]
+bench = false
```

Comment:
Not strictly necessary for this change, but gets a lot of extraneous output out of `cargo bench` invocations even when `--bench parse` is not specified.